lib/generate-index.js

```diff
@@ -38,7 +38,19 @@ function generateIndex (playbook, pages) {
     // Exclude pages marked as "noindex"
     .filter(({ page, $ }) => {
       const $metaRobots = $('meta[name=robots]')
-      return !(($metaRobots && $metaRobots.attr('content') === 'noindex') || (page.asciidoc && page.asciidoc.attributes && page.asciidoc.attributes.noindex === ''))
+      const indexable = !(($metaRobots && $metaRobots.attr('content') === 'noindex') ||
+                        (page.asciidoc && page.asciidoc.attributes && page.asciidoc.attributes.noindex === ''))
```

Comment:
Not really related to your changes but this is hard to read, maybe:

```suggestion
      const metaRobotNoIndex = $metaRobots && $metaRobots.attr('content') === 'noindex'
      const pageNoIndex = page.asciidoc && page.asciidoc.attributes && page.asciidoc.attributes.noindex === ''
      const noIndex = metaRobotNoIndex || pageNoIndex
```